test/1.4/lib/T_attributes.dml

```diff
@@ -30,6 +30,16 @@ attribute f is (double_attr) {
 //    method set(attr_value_t value) throws default { default(value); }
 }
 
+attribute g is (bool_attr) {
+    param init_val = true;
+//  method init() default { val = init_val; }
```

Comment:
remove comment, or reformulate it in English.

The test only covers bool_attr, can we have init_val tests for the other templates as well?

Reply:
> remove comment, or reformulate it in English.

Just followed the pattern of comments in the test. I'd be happy to remove it.

> The test only covers bool_attr, can we have init_val tests for the other templates as well?

Sure.